plugins/producer_plugin/include/eosio/producer_plugin/producer_plugin.hpp

```diff
@@ -158,6 +182,9 @@ FC_REFLECT(eosio::producer_plugin::greylist_params, (accounts));
 FC_REFLECT(eosio::producer_plugin::whitelist_blacklist, (actor_whitelist)(actor_blacklist)(contract_whitelist)(contract_blacklist)(action_blacklist)(key_blacklist) )
 FC_REFLECT(eosio::producer_plugin::integrity_hash_information, (head_block_id)(integrity_hash))
 FC_REFLECT(eosio::producer_plugin::snapshot_information, (head_block_id)(head_block_num)(head_block_time)(version)(snapshot_name))
+FC_REFLECT(eosio::producer_plugin::snapshot_request_information, (block_spacing)(start_block_num)(end_block_num)(snapshot_description))
+FC_REFLECT(eosio::producer_plugin::snapshot_request_id_information, (snapshot_request_id))
+FC_REFLECT(eosio::producer_plugin::get_snapshot_requests_result, (requests))
```

Comment:
I thought you had to use FC_REFLECT_DERIVED for these derived types.

Reply:
done